.github/workflows/integrationTests.yml

```diff
@@ -0,0 +1,65 @@
+name: IntegrationTest
+on:
+  push:
+    branches:
+      - main
+      - dev
+    tags: ['*']
+  pull_request:
+  workflow_dispatch:
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.7'
```

Comment:
Why there is no `1.6`? Shouldn't it be included, because it's the LTS?

Reply:
I fixed it. Now the test project is dynamically created. Therefore also 1.6 works.